app/assets/stylesheets/pages/nodes_list.scss

```diff
@@ -128,10 +138,15 @@
     .failed-message {
       display: inline-block;
     }
+
+    .actions .btn-link {
```

Comment:
> There's already `visibility: hidden` in line 108, there's no need to override it in the failed state. Is this a new condition?

i think the issue was because in case of a failed admin migration, the migrate admin button still showed up in the error message. i couldnt find any js where we remove it in the failed update case, so i found this css code. i was not really knowing what i am doing, just try and error... do you have an idea why the button is visible in the failed admin migration case?

Reply:
The button link will appear if `data.admin.tx_update_migration_available` is `true` or if `data.admin.tx_update_reboot_needed` is true even having both cases failed.

So the requirement is that in case of a failed update/migration, the user won't be able to click on `Update/Migrate admin node` anymore?